pgmpy/inference/ExactInference.py

```diff
@@ -228,6 +229,48 @@ def induced_graph(self, elimination_order):
         >>> inference.induced_graph(['C', 'D', 'A', 'B', 'E'])
         <networkx.classes.graph.Graph at 0x7f34ac8c5160>
         """
+        # If the elimination_order list is incomplete, raise an error
+        if len(elimination_order) < len(self.variables):
```

Comment:
@pratyakshs Why do you need this check when you are already checking for equality in the next line? This check is redundant.